grakn-dist/src/bin/grakn-engine.sh

```diff
@@ -38,6 +38,9 @@ for jar in "${GRAKN_HOME}"/lib/*.jar; do
     fi
 done
 
+# Add path containing logback.xml
+CLASSPATH="$CLASSPATH":"${GRAKN_HOME}"/conf/main/
```

Comment:
This probably should not be hard coded. What if someone specifies their properties and logback conf somewhere else